python/ray/data/read_api.py

```diff
@@ -1786,13 +1798,13 @@ def from_huggingface(
         )
 
     Args:
-        dataset: A Hugging Face Dataset, or DatasetDict. IterableDataset is not
-            supported. ``IterableDataset`` is not supported.
+        dataset: A `Hugging Face Datasets Dataset`_ or `DatasetDict`_.
+            :class:`~ray.data.IterableDataset` is not supported.
 
     Returns:
-        Dataset holding Arrow records from the Hugging Face Dataset, or a dict of
-            datasets in case dataset is a DatasetDict.
-    """
+        A :class:`~ray.data.Dataset` holding Arrow records from the `Hugging Face Datasets Dataset`_,
+        or a dict of :class:`~ray.data.Dataset` s in case ``dataset`` is a `DatasetDict`_.
```

Comment:
```suggestion
        A :class:`~ray.data.Dataset` holding rows from the `Hugging Face Datasets Dataset`_,
        or a dict of :class:`~ray.data.Dataset` s in case ``dataset`` is a `DatasetDict`_.
```